keyboards/sthlmkb/storre/storre.c

```diff
@@ -0,0 +1,36 @@
+/* Copyright 2024 mohoyt
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "quantum.h"
+
+#ifdef ENCODER_ENABLE
```

Comment:
@mohoyt 
Any thought given to using encoder_map in keymap(s)? rather than placing encoder keycodes at keyboard level

Reply:
Added it at keycap level too (and added corresponding `ENCODER_MAP_ENABLE = yes` in rules.mk too)